media/base/supported_types.cc

```diff
@@ -70,7 +71,11 @@ bool IsSupportedHdrMetadata(const gfx::HdrMetadataType& hdr_metadata_type) {
       return true;
 
     case gfx::HdrMetadataType::kSmpteSt2086:
```

Comment:
What's the base of this change?  The implementation of `IsSupportedHdrMetadata()` seems to be different than the current trunk version: http://shortn/_0iLW0ElVnF.

Reply:
A new line was added in m114 https://chromium.googlesource.com/chromium/src/+/refs/tags/114.0.5735.331/media/base/supported_types.cc#72

Reply:
Ah, this change is based on earlier changes on this branch. Here's the initial commits to update media https://github.com/youtube/cobalt/pull/2834

Reply:
After some changes to the feature/media-upstream-update branch, this PR now applies all customizations.